__tests__/endpoint.test.js

```diff
@@ -241,3 +241,61 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+// Tests Task 7:----------------------------
+
+describe("POST: /api/articles/:article_id/comments", () => {
+  test("responds with the inserted comment data", () => {
+    return request(app)
+      .post("/api/articles/9/comments")
+      .expect(201)
+      .send({
+        username: "butter_bridge",
+        body: "I have a body",
+      })
+      .then((response) => {
+        const body = response.body;
+        expect(body).toEqual({
+          username: "butter_bridge",
```

Comment:
comments in the database should contain the following properties: 

 - comment_id
 - votes
 - created_at
 - author which is the username from the users table
- body